src/cli/setup_mode.rs

```diff
@@ -81,6 +81,11 @@ pub fn main() -> Result<utils::ExitCode> {
                 .multiple(true)
                 .use_delimiter(true),
         )
+        .arg(
+            Arg::with_name("no-update-default-toolchain")
```

Comment:
I feel this is a bit verbose. We have --no-self-update; perhaps just --no-update ?

Reply:
I will have a ponder.  `--no-update` feels a little odd but the terseness may be okay.  I'll reconsider in the face of the rest of the args, it is a bit long yes.